static/app/components/idBadge/baseBadge.tsx

```diff
@@ -57,19 +61,13 @@ const BaseBadge = memo(
   )
 );
 
-export default BaseBadge;
-
 const Wrapper = styled('div')`
   display: flex;
+  gap: ${space(1)};
```

Comment:
Idk if this is properly replacing the margin-right that StyledAvatar had before.

The problem I'm seeing is that the replay breadcrumbItem has no spacing now:
![SCR-20240416-jpqv](https://github.com/getsentry/sentry/assets/187460/08f9ee8d-6c4e-49f2-9628-0cffbe27d242)

Reply:
waiting to see the combo with https://github.com/getsentry/sentry/pull/69009, will return to look again!

Reply:
this will have a merge conflict once #69009 merges